File: Basics/DecimaltoBinary.cpp
#include<iostream>

using namespace std;

int main(){
    int number,remainder,answer=0,mul=1;
    cout << "Enter the decimal number to find its binary equivalent :";
    cin >> number;

    while(number != 0){
        remainder = number % 2;
        number /= 2;
        answer = remainder * mul + answer;
        mul = mul * 10;
    }

    cout << "The binary equivalent is : " << answer;
}

File: Basics/DecimaltoBinary.java
public class DecimaltoBinary {
    static int conversion(int number){
        int rem,answer=0;
        while (number != 0) {
            rem = number % 2;
            number/=2;
            answer = answer * 10 + rem;
        }
        return answer;
    }
    public static void main(String[] args) {
        int num = 13;
        System.out.println(conversion(num));
    }
    
}
